addon/const/mime-types.js

```diff
@@ -0,0 +1,449 @@
+/**
+ * @class mimeTypes
+ */
+
+/**
+ * Provides list of all file extensions that can be
+ * rendered as text by MFR
```

Comment:
If this list was retrieved from somewhere else, would be good to provide a link in the docstring here.

Reply:
I got this list from osf.io (https://github.com/baylee-d/osf.io/blob/develop/website/static/js/mime.js).  Should there still be a link?

Reply:
Yeah, but point to this: https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/mime.js

Like I've done here maybe: https://github.com/CenterForOpenScience/ember-osf/pull/240/files#diff-26748c589710c70e6bfe023bc8a3c072R21

Not sure what's the standard way of doing this, but I think if we can't directly import it, we should attribute it

Reply:
Ah, okay that makes sense 👍